test/e2e/tests/test_notebook_instance.py

```diff
@@ -146,7 +150,65 @@ def testCreateAndDelete(self, notebook_instance):
         )
         assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "True")
 
+    def update_notebook_test(self, notebook_instance):
+        (reference, resource, spec) = notebook_instance
+        notebook_instance_name = resource["spec"].get("notebookInstanceName", None)
+        volumeSizeInGB = 7
+
+        # Update test
+        spec["spec"]["volumeSizeInGB"] = volumeSizeInGB
+        # TODO: Use del spec["spec"]["defaultCodeRepository"] instead once ack.testinfra supports replacement.
+        # Patch only supports updating spec fields instead of fully getting rid of them.
+        spec["spec"]["defaultCodeRepository"] = None
+        k8s.patch_custom_resource(reference, spec)
```

Comment:
could we also add an update test where the field did not exist before but is being added in the update ?

Reply:
Added one for additionalCodeRepositories.

Reply:
good!